adapter-http/pom.xml

```diff
@@ -41,6 +41,12 @@
             <version>${revision}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
Can we please do this with Gson instead? jackson keeps creating security and compatibility issues for us across modules. Gson is already in the dependency tree and used for many other SERDES needs.